policies/queries/iam/policies_unused.sql

```diff
@@ -0,0 +1,12 @@
+insert into aws_policy_results
+select :'execution_time'                       as execution_time,
+       :'framework'                            as framework,
+       :'check_id'                             as check_id,
+       'Unused IAM policies should be removed' as title,
+       account_id,
+       id                                      as resource_id,
+       case
+           when attachment_count > 0 or is_attachable = false then 'pass'
```

Comment:
These fields also do not exist... Maybe the table is wrong?

Reply:
aws_iam_policies